content/sifixes/src/header/si/si_faces.uc

```diff
@@ -0,0 +1,223 @@
+	FACE_ERSTAM = 35,
+	FACE_VASEL = 36,
+	// 37 unused
+	FACE_ARGUS = 38,
+	FACE_BYRIN = 39,
+	FACE_HAWK = 40,
+	FACE_DEVRA = 41,
+	FACE_FLINDO = 42,
+	FACE_KANE = 43,
+	FACE_SELINA = 44,
+	FACE_WILFRED = 45,
+	FACE_ALYSSAND = 46,
+	FACE_DELIN = 47,
+	FACE_DELPHYNIA = 48,
+	FACE_GARTH = 49,
+	FACE_JOTH = 50,
+	FACE_VOLDIN = 51,
+	FACE_JENDON = 52,
+	FACE_JORVIN = 53,
+	FACE_KYLISTA = 54,
+	FACE_YELINDA = 55,
+	FACE_LEON = 56,
+	FACE_KALEN = 57,
+	FACE_OLON = 58,
+	FACE_RUGGS = 59,
+	FACE_SCOTS = 60,
+	FACE_ZULITH = 61,
+	FACE_ANDRAL = 62,
+	FACE_CALADIN = 63,
+	FACE_CANTRA = 64,
+	FACE_CELLIA = 65,
+	FACE_G_SIMON = 66,
+	FACE_HARNNA = 67,
+	FACE_KRAYG = 68,
+	FACE_MARSTEN = 69,
+	FACE_LUCILLA = 70,
+	FACE_LUTHER = 71,
+	FACE_LYDIA = 72,
+	FACE_RENFRY = 73,
+	FACE_SHAZZANA = 74,
+	FACE_SHMED = 75,
+	FACE_SIMON = 76,
+	FACE_SPEKTOR = 77,
+	FACE_STANDARR = 78,
+	FACE_TEMPLAR = 79,
+	FACE_FLICKEN = 80,
+	FACE_ENSORCIO = 81,
+	// 82-93 unused
+	FACE_MURDER1 = 94,
+	FACE_MURDER2 = 95,
+	FACE_MURDER3 = 96,
+	FACE_MURDER4 = 97,
+	FACE_MURDER5 = 98,
+	FACE_MURDER6 = 99,
+	// 100-126 unused
+	FACE_BRUNT = 127,
+	FACE_DEADEYE = 128,
+	// 129-142 unused
+	FACE_BAIYANDA = 143,
+	FACE_BWUNDIAI = 144,
+	FACE_MWAERNO = 145,
+	FACE_MYAURI = 146,
+	FACE_NEYOBI = 147,
+	FACE_YENANI = 148,
+	FACE_GWENNO = 149,
+	FACE_BRENDANN = 150,
+	// 151 unused
+	FACE_SETHYS = 152,
+	FACE_GILWOYAI = 153,
+	FACE_KAPYUNDI = 154,
+	FACE_BERYL = 155,
+	FACE_DRAYGAN = 156,
+	// 157-158 unused
+	FACE_MORGHRIM = 159,
+	// 160-167 unused
+	FACE_STEFANO = 168,
+	// 169 unused
+	FACE_ZHELKAS = 170,
+	// 171-177 unused
+	FACE_CLONE_IOLO = 178,
+	FACE_CLONE_SHAMINO = 179,
+	FACE_CLONE_DUPRE = 180,
+	FACE_RABINDRINATH = 181,
+	FACE_SIRANUSH = 182,
+	// 183 unused
+	FACE_MURDER7 = 184,
+	// 185-206 unused
+	FACE_KARNAX = 207,
+	// 208 unused
+	FACE_MIGGIM = 209,
+	//210 unused
+	FACE_THOXA = 211,
+	FACE_BRACCUS = 212,
+	FACE_DRAXTA = 213,
+	// 214 unused
+	FACE_XENKA = 215,
+	// 216-255 unused
+	// Most face numbers used below do not match with NPC numbers.
+	FACE_ARCADION = 256,
+	FACE_BLACK_SWORD = 257,				// 4 frames, only first used in game
+	FACE_SHRIASH = 258,
+	FACE_WISP1 = 259,					// Unused in game
+	FACE_WISP2 = 260,					// Unused in game
+	FACE_ORACLE = 261,
+	FACE_CATMAN = 262,
+	// 263 unused
+	FACE_VASCULIO = 264,
+	// 265 unused
+	FACE_YELINDA_SKINLESS = 266,
+	FACE_SSITHNOS = 267,
+	FACE_ANTI_SHAMINO1 = 268,
+	FACE_FAWN_GUARD = 269,
+	FACE_GOBLIN_KING = 270,
+	FACE_PIKEMAN_BEAR = 271,
+	FACE_SKYRISE = 272,
+	FACE_PALOS = 273,
+	FACE_RANGER = 274,
+	FACE_PIKEMAN_WOLF = 275,
+	FACE_LORD_BRITISH = 276,
+	FACE_GHOST = 277,
+		FRAME_BALD = 0,
+		FRAME_HAIR = 1,
+		FRAME_SMIRK = 2,
+	FACE_PIKEMAN_LEOPARD = 278,
+	FACE_HOUND_OF_DOSKAR = 279,
+	FACE_HAZARD = 280,
+	FACE_ERNESTO = 281,
+	FACE_ANTI_DUPRE1 = 282,
+	// 283 unused
+	FACE_CHAOS_HIEROPHANT = 284,
+	// 285 unused
+	FACE_BEATRIX = 286,
+	FACE_BATLIN = 287,
+		FRAME_WITH_MEDALLION = 0,
+		FRAME_WITHOUT_MEDALLION = 1,
```

Comment:
I think that the FRAME_XXX constants should be renamed to FRAME_<NPCNAME>_XXX. For example, `FRAME_COLUMNA_YOUNG` instead of `FRAME_YOUNG`.

Reply:
Also, maybe move them to a separate enum and remove the double indent.